debian/control

```diff
@@ -34,3 +34,9 @@ Description: management tools for Ubuntu Advantage
  .
  Subscribers to Ubuntu Advantage will find helpful tools for accessing
  services in this package.
+
+Package: ubuntu-advantage-pro
+Architecture: any
+Depends: ubuntu-advantage-tools
+Description: utilities and services for Ubuntu Pro images
```

Comment:
Should we add a conflicts ubuntu-advantage-tools  19.6 since that version of ubuntu-advantage-tools in aws pro already released and has the init scripts in it.

Reply:
Good question. Another way of addressing that would be to have `ubuntu-advantage-pro` depends on the specific version of `ubuntu-advantage-tools` it was built with, which would (I think) force the two to be upgraded in lockstep.

We may want to do that regardless, even if it's a bit of heavy hammer for the specific issue you raised.

@panlinux What do you think?

Reply:
I think we resolved this conversation/question with @basak's review and the subsequent changeset we just pushed.